jekyll-builder-local/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jekyll/builder:4
+FROM jekyll/builder:4.2.0
```

Comment:
The 4.2.2 jekyll builder (published two weeks ago) has some incompatibility with our setup - so pinning this to the latest 4.x that did work